src/ImgixTag.js

```diff
@@ -49,10 +49,14 @@ var ImgixTag = (function() {
   }
 
   ImgixTag.prototype._extractBaseParams = function() {
-    var params;
+		var params = {};
```

Comment:
[nit] Spaces and tabs, my friend... spaces and tabs.

Reply:
🙈 https://www.youtube.com/watch?v=cowtgmZuai0